tests/datatypes/test_bytes.py

```diff
@@ -28,6 +30,12 @@ def test_capitalize(self):
             print(b'2015638687'.capitalize())
         """)
 
+    @unittest.skip("Move this to test_capitalize upon resolution of #530")
```

Comment:
Ohh, got it, that makes sense!
Thanks for clearing that out, sorry for the nitpicking. :)

@katieamazing my bad, I agree w/ @freakboy3742, better to go with expectedFailure here. :+1: